.github/workflows/rebuild-100k.yml

```diff
@@ -0,0 +1,57 @@
+name: Rebuild 100k sample
+
+on:
+  # cron job once a week on mondays
+  schedule:
+    - cron:  '* * * * 1'
+  # Manually triggered using GitHub's UI
+  workflow_dispatch:
+
+jobs:
+  gisaid:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: nextstrain/.github/actions/setup-nextstrain-cli@master
+      with:
+        python-version: 3.9
+
+    - name: Launch build
+      run: |
+        set -x
+
+        declare -a config
+        config+=(slack_token=$SLACK_TOKEN)
+
+        nextstrain build \
+          --aws-batch \
+          --detach \
+          --cpus 16 \
+          --memory 14GiB \
+          . \
+            upload \
+            --configfile nextstrain_profiles/100k/config.yaml \
+            --config "${config[@]}" \
+            --set-threads tree=8 \
+        |& tee build-launch.log
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+
+    - name: Build info
+      run: |
+        echo "--> 100k sample rebuilding on AWS"
+        echo
+        echo "--> When completed, the following 2 files will be updated:"
+        echo "s3://nextstrain-ncov-private/100k/metadata.tsv.xz"
+        echo "s3://nextstrain-ncov-private/100k/sequences.fasta.xz"
```

Comment:
We should either only or additionally use `zst` rather than `xz` it (de)compresses much faster

Reply:
Ah I see, since this runs in ncov rather than ingest this may not be so easy as ncov isn't yet `zstd` ready I suppose.

Reply:
Yeah -- the intermediate files are already xz compressed, this is simply uploading them. zstd would be a bigger change to the pipeline internals.